### PR TITLE
[UPD] Update translation files

### PR DESCRIPTION
**Pull request by yucer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ addons:
 
 env:
   global:
-  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="1"
```
Comment (guewen) [inline .travis.yml:23]: It's expected to be done in the job `  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"`

**Review by guewen — COMMENTED** [review]
If I look in the last build for 12.0, it generated the .pot and had nothing to push on github (https://travis-ci.org/OCA/queue/jobs/607667250#L933-L934)
Can you describe the issue? Are some terms missing in the .pot?

**Comment by yucer** [comment]
I didn't see a `.pot` file at all (https://github.com/OCA/queue/tree/12.0/queue_job_subscribe/i18n)

**Comment by guewen** [comment]
Oh, I looked in queue_job. You are right, but the module is not migrated in 12.0 (https://github.com/OCA/queue/blob/d2bf932e94ce0375fd9fa59f87dd3f066a934557/queue_job_subscribe/__manifest__.py#L17)

**Comment by guewen** [comment]
Same as #193

**Pull request closed without merging**